airflow/providers/jira/sensors/jira.py

```diff
@@ -17,9 +17,10 @@
 # under the License.
 from typing import Any, Callable, Dict, Optional
 
+from jira.exceptions import JIRAError
 from jira.resources import Issue, Resource
 
-from airflow.providers.jira.operators.jira import JIRAError, JiraOperator
+from airflow.providers.jira.operators.jira import JiraOperator
```

Comment:
What was the sort of error this gave with the import as it was?

Reply:
```
WARNING: Cannot resolve import of airflow.providers.jira.operators.jira.JIRAError in airflow.providers.jira.sensors.jira
```